main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+)
+
+func createPhoneBook(names []string, numbers []string) map[string]string {
+	x := make(map[string]string)
+	for a, b := range names {
+		x[b] = numbers[a]
+	}
+	return x
+}
+
+func main() {
+	var c int
+	fmt.Print("How many contacts do you want to enter?")
+	fmt.Scan(&c)
+	var names, numbers []string
+	var scan string
+	for y := 1; y <= c; y++ {
+		fmt.Printf("Enter name #%v:", y)
+		fmt.Scan(&scan)
```

Comment:
`fmt.Scan` doesn't work with strings containing spaces. You need to use `bufio.NewReader(os.Stdin)` in order to be able to read string with spaces.